frontend/src/api/assignments.ts

```diff
@@ -49,3 +49,29 @@ export const getAssignmentIndirectNav = async (
 
   return data;
 };
+
+
+
+export const acceptAssignment = async (orgName: string, repoName: string, classroomID: number, assignmentName: string) => {
+  const result = await fetch(
+      `${base_url}/classrooms/classroom/${classroomID}/assignments/accept`,
+      {
+        method: "POST",
+        credentials: "include",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          org_name: orgName,
+          repo_name: repoName,
+          assignment_name: assignmentName,
+          assignment_id: 1,
+          org_id: 182810684
```

Comment:
I assume this is mock data since the assignment stuff isn't done yet? Or was this left unintentionally?

Reply:
Yea it's mocking other data we will pull from the assignment creation / acceptance flow